sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeSentimentAsyncClient.java

```diff
@@ -86,16 +97,18 @@
      * which contains {@link AnalyzeSentimentResultCollection}.
      *
      * @param documents The list of documents to analyze sentiments for.
+     * @param includeOpinionMining The boolean indicator to include opinion mining data in the returned result.
```

Comment:
Should include that if this flag is specified, you'll get a `minedOpinions` property on `SentenceSentiment`

Reply:
Should also include it's only available for v3.1-preview.1

Reply:
thanks.